### PR TITLE
DEP-95: SA Page

### PR DESCRIPTION
**Pull request by oc-ste**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -1,2 +1,13 @@
 #global values
-global.nav.title=Ask HMRC - Webchat
\ No newline at end of file
+global.nav.title=Ask HMRC - Webchat
+
+default.title=Webchat
+default.p1=???
```
Comment (daniel-manning) [inline conf/messages]: Are you waiting on something back?

File: app/uk/gov/hmrc/depfrontend/views/govuk_wrapper.scala.html [hunk 2]
```diff
@@ -58,6 +58,16 @@
...
+
+    <script type="text/javascript">
+            var nuanceData =
+                    {
+                        'sessionId': 'MY-SESSION',
+                        'sessionID': 'MY-SESSION'
+                    }
+    </script>
+
+    <script id="webchat-tag" type="text/javascript" src="https://hmrc-uk.digital.nuance.com/chatskins/launch/inqChatLaunch10006719.js"></script>
```
Comment (daniel-manning) [inline app/uk/gov/hmrc/depfrontend/views/govuk_wrapper.scala.html]: pull the url out into appconfig

File: conf/messages (hunk 1 above)
Comment (oc-ste) [reply conf/messages]: Yeah, no content for a default page yet

**Comment by oc-ste** [comment]
No Welsh at all as far as I'm aware.

**Pull request merged**